### PR TITLE
Sanic does not support Python 3.5 and won't need this code.

### PR DESCRIPTION
**Pull request by Tronic**

### CODE REVIEW

File: sanic/request.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import asyncio
 import email.utils
 import json
```
Comment (abuckenheimer) [inline sanic/request.py]: minor tweak, we don't need to import json if ujson doesn't exist, we should make this import in the except branch

Comment (Tronic) [reply sanic/request.py]: Good catch!

**Comment by Tronic** [comment]
Fixing bugs while riding the metro, without testing... Doesn't always work!

**Comment by Tronic** [comment]
You can do the squash by pressing the arrow button next to the merge button, if I am not mistaken.

**Pull request merged**